app/retail/views.py

```diff
@@ -459,6 +463,88 @@ def browser_extension_chrome(request):
 def browser_extension_firefox(request):
     return redirect('https://addons.mozilla.org/en-US/firefox/addon/gitcoin/')
 
+def idea_show(request, idea_id):    
+    # params = {
+    #     'active': 'dashboard',
+    #     'title': 'Issue Explorer',
+    #     'keywords': json.dumps([str(key) for key in Keyword.objects.all().values_list('keyword', flat=True)]),
+    # }    
+    return TemplateResponse(request, 'idea.html', {})
+
+@csrf_exempt
+def new_idea(request):
+
+    # params = {
+    #     'active': 'dashboard',
+    #     'title': 'Issue Explorer',
+    #     'keywords': json.dumps([str(key) for key in Keyword.objects.all().values_list('keyword', flat=True)]),
+    # }
+    params = {}
+    return TemplateResponse(request, 'new_idea.html', params)
+
+@require_POST
+@csrf_exempt
+def create_idea(request):
+    idea = Idea(
+        full_name=request.POST.get('fullName'),
+        email=request.POST.get('email'),
+        github_username=request.POST.get('githubUsername'),
+        summary=request.POST.get('summary'),
+        more_info=request.POST.get('moreInfo'),
+        looking_for_capital=True, #request.POST.get('lookingForCapital') returns 'fasle' or 'true'
```

Comment:
I have no idea how to serialize properly javascript's true/false to python's True/False, any ideas?

Reply:
Per https://stackoverflow.com/questions/10693630/how-to-pass-a-boolean-from-javascript-to-python

Reply:
Per https://stackoverflow.com/questions/10693630/how-to-pass-a-boolean-from-javascript-to-python

Reply:
Per https://stackoverflow.com/questions/10693630/how-to-pass-a-boolean-from-javascript-to-python

Reply:
Per [this](https://stackoverflow.com/questions/10693630/how-to-pass-a-boolean-from-javascript-to-python )

> looking_for_capital = True if request.POST.get('lookingForCapital') == 'true' else False
or
> looking_for_capital = request.POST.get('lookingForCapital') == 'true'

might be a good solution 🤔

Reply:
Per [this](https://stackoverflow.com/a/10693668)

> looking_for_capital = True if request.POST.get('lookingForCapital') == 'true' else False

or

> looking_for_capital = request.POST.get('lookingForCaptail') == 'true'

Reply:
Yeah i saw this solution, i wonder if there is another...

Reply:
I mean framework should provide proper deserializer, no manual checks spread across the code

Reply:
Imma try this tomorrow: https://stackoverflow.com/questions/15476983/deserialize-a-json-string-to-an-object-in-python

Reply:
Neat that looks a lot cleaner 👍🏻